### DISCUSSION TITLE
Distribution Packages: What do you use? Would you like to help?

### DISCUSSION THREAD

**Discussion by bdd** [bdd/runitor#40] [Polls]
I'd like to invest some time into getting runitor into some Linux distributions' package index.

It's already in NixOS 22.05 (and also Nixpkgs for macOS)
I intend to make an Alpine package too. It seems to be rather popular as the base image for most containers.
I do NOT plan to provide a Docker image if it makes it into Alpine.

## Survey
- Help me understand under what operating environments you use runitor by commenting here.
- Would you'd find having up-to-date platform native packages helpful, or are you happy with building it yourself or downloading from GitHub?

## Not Linux
- Do you use runitor under Windows? If so, what is you preferred way getting software packages?
- Any macOS users? Would you like to see it in Homebrew?

## Help Wanted
I can use help building and maintaining packages for:
* Debian (extra important for Raspberry Pi users out there)
* FreeBSD
* OpenBSD

**Comment by cuu508** [comment]
I use runitor on Ubuntu (x86-64) systems. I have provisioning scripts for setting systems up. The provisioning scripts upload a runitor binary (downloaded from GitHub Releases page) to `/usr/local/bin/runitor`. It works fine, but it would be awesome to be able to do `apt install runitor` instead.
